PSReadLine.build.ps1

```diff
@@ -199,15 +219,26 @@ task LayoutModule BuildMainModule, BuildMamlHelp, {
         Copy-Item $file $targetDir
     }
 
-
-    Copy-Item PSReadLine/bin/$Configuration/Microsoft.PowerShell.PSReadLine2.dll $targetDir
-    Copy-Item PSReadLine/bin/$Configuration/System.Runtime.InteropServices.RuntimeInformation.dll $targetDir
```

Comment:
Why did you remove copying `System.Runtime.InteropServices.RuntimeInformation.dll`? Is this dll installed everywhere PSReadLine including Win7?